docs/user/dql/functions.rst

```diff
@@ -2164,6 +2164,31 @@ Example::
     +--------------------------------------+
 
 
+STR_TO_DATE
+-----------
+
+Description
+>>>>>>>>>>>
+
+Usage: str_to_date(string, string) is used to extract a DATE, TIME, or DATETIME from the first argument string using the formats specified in the second argument string.
+Acceptable string format specifiers are the same as those used in the `DATE_FORMAT`_ function.
+Does return NULL when a statement cannot be parsed due to an invalid pair of arguments, and when 0 is provided for any field.
+
+Argument type: STRING, STRING
+
+Return type: DATE, DATETIME, TIME (dependent on the format string provided as the second argument)
```

Comment:
```suggestion
Return type: DATETIME
```

Reply:
Mention (copy from another func) how time/date converted to datetime

Reply:
Fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/231/commits/0ab10018ddbb44e181e89883e8c69b16825daa05